CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 3.5.1 (2020-04-08)
+
+- Added a deprecation message to the `SkyPopoverTrigger` type. [#63](https://github.com/blackbaud/skyux-popovers/pull/63)
+- Fixed the dropdown component to properly set all ARIA attributes. [#67](https://github.com/blackbaud/skyux-popovers/pull/67)
+- Fixed `SkyPopoverContext` to not throw build warnings. [#63](https://github.com/blackbaud/skyux-popovers/pull/63)
```

Comment:
Would "to stop" or "to prevent" be acceptable? I just find that "to not" can sometimes read a little funny.